packages/block-library/src/image/index.js

```diff
@@ -30,15 +30,9 @@ export const name = 'core/image';
 const blockAttributes = {
 	url: {
 		type: 'string',
-		source: 'attribute',
-		selector: 'img',
-		attribute: 'src',
```

Comment:
This change will invalidate all existing image blocks.

![image](https://user-images.githubusercontent.com/1779930/48784390-97cd1800-ecb0-11e8-9bcd-6b00f363067b.png)

```
Block validation: Block validation failed for `core/image` ({name: "core/image", title: "Image", description: "Insert an image to make a visual statement.", icon: {…}, category: "common", …}).

Expected:

<figure class="wp-block-image alignwide"><img alt="" class="wp-image-1962"/></figure>

Actual:

<figure class="wp-block-image alignwide"><img src="http://editor.test/wp-content/uploads/2018/11/desk-1024x614.jpeg" alt="" class="wp-image-1962" /></figure>
```

https://wordpress.org/gutenberg/handbook/block-api/deprecated-blocks/

Reply:
Yeah, need to add perhaps `oldBlockAttributes` for use in the deprecated save().